scripts/consume_sitemaps.py

```diff
@@ -186,7 +186,6 @@ def fetch_sitemap_files(authority, last_updated_date, thing_ids: typing.Dict[str
                             thing_dict["primary_key"] = thing_ids[thing_identifier]
                             current_existing_things_batch.append(thing_dict)
                         else:
-                            thing_dict["tcreated"] = now
```

Comment:
This was nuking the previous code that we added that actually tracked the last mod time.  Whoopsie.